packages/strapi-plugin-content-manager/admin/src/components/CustomTable/Row.js

```diff
@@ -3,9 +3,12 @@ import { withRouter } from 'react-router';
 import PropTypes from 'prop-types';
 import { get, isEmpty, isNull, isObject, toLower, toString } from 'lodash';
 import moment from 'moment';
-import { IcoContainer, useGlobalContext } from 'strapi-helper-plugin';
+import {
+  IcoContainer,
+  useGlobalContext,
+  dateFormats,
```

Comment:
Use the file override file for dateFormats so users are still able to customise it easily.